language-support/java/codegen/BUILD.bazel

```diff
@@ -115,6 +116,19 @@ dar_to_java(
     package_prefix = "ut",
 )
 
+daml_compile(
+    name = "test-interface-retro-implements-daml",
+    srcs = ["src/test/daml/Retro.daml"],
```

Comment:
I think you can extend the existing `test-contract-id-daml` with this new source file instead; that would probably build faster, since the relevant test will already have access.